src/main/java/org/ihtsdo/otf/traceabilityservice/service/ReportService.java

```diff
@@ -109,13 +108,6 @@ private void processChangesRebasedToBranch(String branch, Long contentHeadTimest
 												.gt(startDate.getTime())
 												.lte(contentHeadTimestamp != null ? contentHeadTimestamp : previousLevelBaseDate.getTime())));
 				processCommits(onAncestorBranch, componentChanges, changesNotAtTaskLevel, componentToConceptIdMap);
-
-				previousLevel = ancestor;
-				if (BranchUtil.isCodeSystemBranch(ancestor)) {
-					// Stop at any code system level
-					// We don't expect to inherit unversioned content from other code systems.
-					break;
-				}
```

Comment:
We are starting at the code system level and going down, therefore, this break at the start is not required.